src/components/calcite-fab/calcite-fab.tsx

```diff
@@ -0,0 +1,106 @@
+import { Component, Element, Host, Method, Prop, h } from "@stencil/core";
+import { CalciteTheme } from "../interfaces";
+import { CSS } from "./resources";
+import { focusElement, getElementDir } from "../utils/dom";
+import { plus24F } from "@esri/calcite-ui-icons"; // todo: will remove once icon string supported in `calcite-button`
+
+@Component({
+  tag: "calcite-fab",
+  styleUrl: "calcite-fab.scss",
+  shadow: true
+})
+export class CalciteFab {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * When true, disabled prevents interaction. This state shows items with lower opacity/grayed.
```

Comment:
Nitpick: Do we need to describe the disabled look (`This state shows items with lower opacity/grayed.`)?

Reply:
I'm not sure. Question for @kat10140 . There are other files with the same text. 
![image](https://user-images.githubusercontent.com/1231455/74862133-6fdbc280-5300-11ea-9548-761c438da4d6.png)

Reply:
Going to leave as is for now. If you think this text should be updated we can open an issue.